rotkehlchen/types.py

```diff
@@ -400,6 +400,13 @@ def __str__(self) -> str:
             return 'Ethereum Staking'
         return super().__str__()
 
+    def serialize(self) -> str:
+        """
+        Serialize is used expose the enum to the frontend. For consistency we expose the key that
+        that is used in the backend that is compatible with the default deserialize method.
+        """
+        return self.get_key()
```

Comment:
It may make sense to call self.value.lower() directly mentioning it's the same as `.get_key()` as you already do in the docstring but also that you call it directly to avoid an extra function call since serialize() is used a lot.

Or ... we can just push this as a potential optimization in the future if needed